templates/install.tmpl

```diff
@@ -267,6 +267,20 @@
 								<input id="_no_reply_address" name="no_reply_address" value="{{.no_reply_address}}">
 								<span class="help">{{.i18n.Tr "install.no_reply_address_helper"}}</span>
 							</div>
+							<div class="inline field">
+								<label for="password_algorithm">{{.i18n.Tr "install.password_algorithm"}}</label>
+								<div class="ui selection dropdown">
+									<input id="_password_algorithm" type="hidden" name="password_algorithm" value="{{.password_algorithm}}">
```

Comment:
```suggestion
									<input id="password_algorithm" type="hidden" name="password_algorithm" value="{{.password_algorithm}}">
```

Reply:
Hm, I see we underscore in other spots too...why do we do that? I thought label `for` attribute was aimed at id

Reply:
Hmm so I was matching another I'd on the install page with a similar preceding _

Reply:
I tested this pull with it and did work ...

Reply:
I'll drop it it seems inconsistent

Reply:
> The for attribute of <label> must be equal to the id attribute of the related element to bind them together. A label can also be bound to an element by placing the element inside the <label> element. 

So it seems like `no_reply_address` may also be incorrect. If it works when you click on the label and it focuses the input I suppose no harm. Not sure why that works, though, since the label is outside the input.